cli/root.go

```diff
@@ -16,10 +16,15 @@ var rootCmd = &cobra.Command{
 func Execute() error {
 	rootCmd.AddCommand(get)
 	rootCmd.AddCommand(set)
+	rootCmd.AddCommand(server)
+	server.Flags().StringP("port", "o", "", "port you use to start server")
+	server.Flags().StringP("path", "a", "", "path you use to store data")
 	set.Flags().StringP("key", "k", "", "key you use to identify data")
 	set.Flags().StringP("value", "v", "", "data you pass ")
 	set.Flags().StringP("cipher-key", "p", "", "cipher-key you use to make your data secret ")
+	set.Flags().StringP("path", "a", "", "path you use to store data")
```

Comment:
?

Reply:
i dont really understand whats wrong with this line.

Reply:
Nothing much, just don't not quite get so far on how you're making decisions. I'm just in the process of getting used to it :)

<img width="500" alt="Screenshot 2023-04-01 at 16 06 45" src="https://user-images.githubusercontent.com/2529842/229294032-72765138-306e-4819-91e1-4be90d8d5d4d.png">